printer_mods/Long/README.md

```diff
@@ -1,4 +1,4 @@
-Longs Toolhead Mods
-
-Mantis Dual 5015 Toolhead:
+**Longs Toolhead Mods**
+  
+**Mantis Dual 5015 Toolhead:**  
 https://github.com/mandryd/VoronUsers/tree/master/printer_mods/Long/Mantis_Dual_5015
```

Comment:
if you put a link in here, please make it relative

    [Mantis Dual 5015 Toolhead](Mantis_Dual_50515)